### PR TITLE
Release pg_auto_failover-enterprise 1.0.3

### PR DESCRIPTION
**Pull request by mtuncer**

### CODE REVIEW

**Comment by thanodnl** [comment]
Last time I tried installing the debian packages for enterprise there was a problem with the dependencies. I don't see any changes for that now. Did you test installing the packages after building them locally, and does that work?

**Comment by mtuncer** [comment]
No I did not test debians
Let me do that

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,24 @@
+pg-auto-failover-enterprise (1.0.3) testing; urgency=low
+
+  * Fix Terminate conflicting queries at failover time (#194)
```
Comment (thanodnl) [inline debian/changelog]: debian changelogs should not contain the changelog of the application, but instead the changelog to the packaging. I have made that mistake in the past as well :).

Hence the boring changlog here that reads something like `Official release of <package> <version>`

**Review by thanodnl — CHANGES_REQUESTED** [review]

**Comment by mtuncer** [comment]
well I was not able to install pg_auto_failover package via dpkg -i. It complained about missing dependencies.
I run `apt install --fix-broken` after the failure at it installed fine.

**Comment by thanodnl** [comment]
Ok, let me check and reproduce this on debian

**Review by thanodnl — APPROVED** [review]

**Pull request merged**